packages/jsii-calc/test/assembly.jsii

```diff
@@ -137,6 +137,17 @@
     "url": "https://github.com/aws/jsii.git"
   },
   "schema": "jsii/0.10.0",
+  "submodules": {
+    "jsii-calc.DerivedClassHasNoProperties": {},
+    "jsii-calc.InterfaceInNamespaceIncludesClasses": {},
+    "jsii-calc.InterfaceInNamespaceOnlyInterface": {},
+    "jsii-calc.composition": {},
+    "jsii-calc.submodule": {},
+    "jsii-calc.submodule.back_references": {},
+    "jsii-calc.submodule.child": {},
+    "jsii-calc.submodule.nested_submodule": {},
+    "jsii-calc.submodule.nested_submodule.deeplyNested": {}
```

Comment:
I think empty objects are a bad way to model something. Do we really need those here?

Reply:
Why do you want instead? Those would eventually contain the submodule-specific configuration block... I don't want to resort to type unions (`true | SubmoduleConfiguration`?), and if I put `undefined` as a value, then the keys will be missing from the serialized form...